files/hipchat.rb

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+
+require "#{File.dirname(__FILE__)}/base"
+#require_relative 'base'
+
+class Hipchat < BaseHandler
+  def handle
+    timeout_and_retry do
+      case check_data['status'].to_i
```

Comment:
Can you switch on `@event['action']` instead? Switching based on the status was an original mistake we made and I don't want to see it happen again. (see #49)